src/lib/battery/battery.cpp

```diff
@@ -97,6 +97,15 @@ Battery::Battery(int index, ModuleParams *parent, const int sample_interval_us,
 	_param_handles.bat_avrg_current = param_find("BAT_AVRG_CURRENT");
 
 	updateParams();
+
+	// Internal resistance estimation initializations
+	_RSL_est(0, 0) = 4.2f * _params.n_cells;
+	_RSL_est(1, 0) = 0.005f * _params.n_cells;
+	_estimation_covariance(0, 0) = 10.f;
```

Comment:
why 10? Add a comment for every "magic" value here please. E.g. // initialize internal resistance state with 0.005Ohm.

Reply:
Even better if you define these values as constants, e.g.
`static constexpr float  R_DEFAULT = 0.005f;`